Scripts/demand/trips.py

```diff
@@ -210,7 +210,12 @@ def _get_probs(self, age: str, is_car_user: bool) -> pandas.DataFrame:
         return pandas.DataFrame(probs).to_numpy().cumsum(axis=1)
     
     def calculate_car_ownership(self, impedance):
-        acc_purpose = self.purpose_dict["hb_leisure"]
+        try:
```

Comment:
Do I read correctly that car ownership is calculated only with >1 iterations? That would be ok as well, just to know.

Reply:
No, it means that car-ownership model is run only if the demand model has a tour purpose called "hb_leisure", for which it can perform the accessibility calculation. The long-distance demand model does not have such a purpose, so then we skip the whole thing.